tests/jballerina-unit-test/src/test/resources/test-src/types/regexp/regexp_type_test.bal

```diff
@@ -105,6 +106,9 @@ function testRegExpWithUserDefinedType() {
     assertEquality(re `AB*|[^c-d]{1,5}`, x7);
 }
 
+type T1 string:RegExp & readonly;
+type T2 regexp:RegExp & readonly;
```

Comment:
Shall we add a test for a local variable of such an intersection type too. Group it with this and use an appropriate name to indicate what we are testing.

Reply:
Updated in [dcf7af](https://github.com/ballerina-platform/ballerina-lang/pull/41843/commits/dcf7afbfeff44123692ed2e30c555553bdba2ec7)